packages/go_router_builder/example/lib/main.dart

```diff
@@ -98,8 +96,7 @@ class LoginRoute extends GoRouteData {
   final String? fromPage;
 
   @override
-  Widget build(BuildContext context, GoRouterState state) =>
-      LoginScreen(from: fromPage);
+  Widget build(BuildContext context, GoRouterState state) => LoginScreen(from: fromPage);
```

Comment:
we use `dart format .` on this project directly, this may be causing repro-check failures. Can you run `dart format` to reformat these files?